10up/simple-local-avatars
# Feature/add crop screen

### Description of the Change

- This PR attempts to add a crop screen after selecting an Avatar Image. Currently the suggested image dimensions: 200 by 200 pixels. 
- The crop screen is skipped if the Avatar size is  200 by 200 pixels.
- `Select Avatar` CTA is renamed to `Select Avatar and Crop`.
- The code is updated to ES6 standards.

### Screenshots

<img width="1736" alt="Screenshot 2021-09-29 at 4 49 07 PM" src="https://user-images.githubusercontent.com/9847340/135258802-e5f5b06a-b3cb-4698-bfa0-75e976e35af7.png">


### Alternate Designs

None

### Benefits

Now the users can crop their Avatar as they want it to appear on the website.

### Possible Drawbacks

Currently i havent used the `api.CroppedImageControl` and `api.SiteIconControl` but these can be used, in order to make the code more robust.

### Verification Process

1. Visit the user profile screen and scroll to the Avatar section at the bottom of the page.
2. Try updating/adding the Avatar image and you should be able to crop the Avatar Image as per your choice.

### Checklist:






### Applicable Issues

https://github.com/10up/simple-local-avatars/issues/65

### Changelog Entry
